docs/docs/02-admins/01-architecture/01-services.md

```diff
@@ -0,0 +1,69 @@
+# Services
+
+A summary of the main services that make up Renku and how they are organized.
+
+## Overview
+
+Unless specifically stated every service below is either a Deployment or StatefulSet
+in Kubernetes.
+
+### Developed in-house
+
+- **Gateway**: Login, routing and injecting Keycloak access tokens.
+- **Data service**: The main backend service for Renku.
+- **Secret service**: Stores and manipulates user secrets.
+- **K8s watcher**: Watches Kubernetes events for user sessions and populates the information in Postgres.
+- **UI**: Serves static assets for the user interface.
```

Comment:
Not anymore now that we can do SSR which means that the UI is not made of only static assets now.

Reply:
I was trying to say that it is just a file server essentially. Is that not true? I guess it has the few endpoints that serve the configuration the ui needs. So I can say it is "a simple express  server for configuration and frontend assets". Is that better?